### DISCUSSION TITLE
Error when opening notifications

### DISCUSSION THREAD

**Discussion by efb4f5ff-1298-471a-8973-3d47447115dc** [community/community#43301] [Mobile]
Hi everytime i open the notification tab i see this error.

![Screenshot_20230104-135819.png](https://user-images.githubusercontent.com/73130443/210560196-232524d4-237e-4744-986a-187bc7efb739.png)

**Comment by HenSquared** [comment]
Hi @efb4f5ff-1298-471a-8973-3d47447115dc,

Thanks for the feedback! If you have access to an iOS device, I wonder if this also happens to your user on iOS devices.

We'll take a look at this and follow up with you!

**Reply by efb4f5ff-1298-471a-8973-3d47447115dc** [reply]
Hi @HenSquared unfortunately i dont have an iOS device. So im unable to isolate if this is an Android only case.

**Reply by efb4f5ff-1298-471a-8973-3d47447115dc** [reply]
Hi @HenSquared 
I have found a workaround

open notifications tab in github app -> see error -> refesh to check if phone is not acting up -> opening my laptop and logging into github in the browser -> github notices that there is activity on my account -> refresh on phone -> see error is gone

Also some more info about the bug: it seems like it throws this error after long periods of inactivity. For example when i go to bed and sleep for 8hrs.

**Reply by HenSquared** [reply]
Thanks a lot for this workaround. I will add this context to our issue tracking this.

**Reply by alcere** [reply]
@efb4f5ff-1298-471a-8973-3d47447115dc We have recently shipped some changes, and we believe we have fixed this issue. San you double check if this is still happening? Thanks!

**Reply by efb4f5ff-1298-471a-8973-3d47447115dc** [reply]
Hi @alcere it seems like it is working now!

**Comment by dato** [comment]
I'm not sure if it's the same issue, but I get something similar on iOS 16.3:

![image](https://user-images.githubusercontent.com/73768/221307388-728c1c08-7526-49fe-a182-0830e2434404.jpeg)

Something to note: if I restart the app and open notifications, for a moment I can see _old_ notifications (the newest one from 5 days ago), then it immediately shows the error, without having to tap on anything.

Also: I didn't try the workaround exactly as described, but I installed the app on a new device, that asked me for my login details, the result was the error above too.

Thanks!

**Reply by dato** [reply]
Forgot to add, in case it could be relevant: push notifications show up in my lock screen just fine.

**Reply by HenSquared** [reply]
@dato, thanks for reporting this. It sounds like a similar issue. I'll add this report to our internal tracking issue and hope that helps us find the root cause.

**Reply by alcere** [reply]
@dato We have recently shipped some changes, and we believe we have fixed this issue. San you double check if this is still happening? Thanks!

**Reply by dato** [reply]
Yes, it works reliably now, thank you.

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
Hi @HenSquared i noticed something a while back but i forgot to inform u about it.

The workaround allows me to see my **unread** notifications but sometimes it doesnt allow me to see my all of them when i reset the filter

https://user-images.githubusercontent.com/73130443/223379847-7c063004-1fb2-4580-bdac-4bf330f9569a.mp4

By default my notifications are filtered on unread on GH web so my hypothesis is if the workaround works for unread notifications it must also work for read notifications.

So what i will try is to go to my inbox and open some read notifications and check if this solves it.
